src/routes/$repo.$.tsx

```diff
@@ -2,7 +2,8 @@ import { RateReview as ReviewIcon } from "@styled-icons/material"
 import { resolve } from "path"
 import { useState } from "react"
 import { useBoolean } from "react-use"
-import { ActionFunction, ErrorBoundaryComponent, json, Link, LoaderFunction, redirect, useLoaderData } from "remix"
+import { ActionFunction, ErrorBoundaryComponent, json, LoaderFunction, redirect } from "@remix-run/node";
```

Comment:
eslint would like to use import type

Reply:
I have now fixed this issue across the codebase by running `npm run lint`